__tests__/app.test.js

```diff
@@ -87,4 +88,45 @@ describe("GET/api/articles/:article_id",()=>{
         expect(body.msg).toBe("not found")
     })
     })
+})
+
+describe("GET /api/articles",()=>{
+    test("responds with 200 and all articles in an array",()=>{
```

Comment:
We need to check that the body property is not included in each article - the toMatchObject matcher will ignore any additional properties on the object besides the mentioned ones. Therefore if the body property is in each article, this test will still pass